docs/quickstart.rst

```diff
@@ -17,8 +17,9 @@ To get started, you'll need to first `install Docker and get it running <https:/
 Running Confluent Platform on Docker
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you're running on Windows or Mac OS X, you'll need to use `Docker Machine <https://docs.docker.com/machine/install-machine/>`_ to start the Docker host.  Docker runs natively on Linux, so the Docker host will be your local machine if you go that route.  If you are running on Mac or Windows, be sure to allocate at least 4 GB of ram to the Docker Machine.
-
+  .. Important::
```

Comment:
I think we should leave this, as it may still apply.

Rather, here's the scary section we need to replace:

https://github.com/confluentinc/cp-docker-images/blob/3.3.x/README.md#important-caveat---images-not-supported-for-docker-for-mac